scss/titles.scss

```diff
@@ -0,0 +1,24 @@
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+
+.title:before{
+    background: url(assets/triangle.svg);
+    background-repeat: no-repeat;
+    content: '';
+    height: $title-decoration-height;
+    position: absolute;
+    right: calc(100% - 20px);
+    top: 50%;
+    transform: translateY(-50%);
+    width: $title-decoration-width;
+    z-index: -1;
+}
+
+.title {
+    color: $mine-shaft;
+    font-family: $font-stack;
+    font-size: $text-big;
+    line-height: 24px;
+    margin: 34px 0 34px 50px;
+    position: absolute;
```

Comment:
remember this should have position relative to be the origin of the absolute positioned child.